src/components/BrandConfigProvider/theme/ThemeProvider.tsx

```diff
@@ -21,6 +21,7 @@ const combinedDefaultTheme = merge(
   MD3DefaultTheme,
   baseDefaultTheme,
 );
+Object.assign(DefaultTheme, combinedDefaultTheme); // Set a new "default" theme
```

Comment:
Hopefully @namick sees / comments on this before merging, to make sure we're not going in the wrong direction here.  Makes sense to me.

Reply:
Good offline conversation with @bruce-glazier, with one minor change given that DefaultTheme and MD3LightTheme are[ actually the same](https://github.com/callstack/react-native-paper/blob/main/src/core/theming.tsx#L19) theme - we're all on the same page and this is the right direction!